docs: documented pr review and approve in e2e-tests (#707)
redhat-appstudio/e2e-tests: README.md

There is a mage target that can cleanup those repositories - `mage local:cleanupGithubOrg`.

For more infor & usage, please run `mage -h local:cleanupGithubOrg`.

# PR review process in e2e-tests
* Both [CODEOWNERS](https://github.com/redhat-appstudio/e2e-tests/blob/main/.github/CODEOWNERS) file and [OWNERS files](https://github.com/redhat-appstudio/e2e-tests/blob/main/OWNERS) (in a directory or its parent directory) take effect in e2e-tests PR review process.

>   * Code owners are automatically requested for review when open a PR. While openshift-ci bot also request a review according to the set of OWNERS files nearest to the code being changed.

Reviewer: ```suggestion
  * Code owners are automatically requested for review when a PR is opened, while openshift-ci bot also requests a review according to the set of OWNERS files nearest to the code being changed.
```

>   * [CODEOWNERS](https://github.com/redhat-appstudio/e2e-tests/blob/main/.github/CODEOWNERS) and root [OWNERS](https://github.com/redhat-appstudio/e2e-tests/blob/main/OWNERS) are maintained by the whole RHTAP QE team. While sub directory OWNERS files are maintained by sub QE team.

Reviewer: ```suggestion
  * [CODEOWNERS](https://github.com/redhat-appstudio/e2e-tests/blob/main/.github/CODEOWNERS) and root [OWNERS](https://github.com/redhat-appstudio/e2e-tests/blob/main/OWNERS) are maintained by the whole RHTAP QE team, while sub directory OWNERS files are maintained by sub QE team.
```

> * To get a PR merged, following is needed:

Reviewer: ```suggestion
* To get a PR merged, the following is needed:
```

>   1. Approval via GitHub UI(Review changes -> select Approve -> Submit review). This approval is valid only from repository collaborators (with Triage permissions or higher).

Reviewer: ```suggestion
  1. Approval via GitHub UI (Review changes -> select Approve -> Submit review). This approval is valid only from repository collaborators (with Triage permissions or higher).
```

Reviewer 2: I'm fine with this PR once Simon's comments are fixed
